test/CMakeLists.txt

```diff
@@ -395,6 +395,24 @@ if (HDF5_BUILD_UTILS)
   set (H5_TESTS ${H5_TESTS} mirror_vfd)
 endif ()
 
+set (HDF5_API_TESTS
```

Comment:
This should be in a separate CMakeAPITests.cmake file that can be guarded by the option value.

Reply:
I would think it would be just as simple to guard the option here rather than creating yet another file just for these tests. These tests aren't particularly special and should be run with the native VOL connector as well.